scripts/kconfig/kconfig.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+# Modified from: https://github.com/ulfalizer/Kconfiglib/blob/master/examples/merge_config.py
```

Comment:
And thus ISC license as well? Very small script so perhaps less of a concern?

Reply:
Yes, ISC as well, so that is not an issue.